practice/src/javascripts/main.js

```diff
@@ -0,0 +1,27 @@
+import ProductList from './products/productList';
+
+window.addEventListener('DOMContentLoaded', () => {
+  const productList = new ProductList();
```

Comment:
Should new ProductService and ProductTemplate here. Then add it as a argument to ProductList

ProductList(service, template)